server/test/unit/server/db/migrations/test_0027_importer_schema_change.py

```diff
@@ -8,7 +8,7 @@
 LAST_SYNC = 'last_sync'
 LAST_UPDATED = 'last_updated'
 LAST_OVERRIDE_CONFIG = 'last_override_config'
-MIGRATION = 'pulp.server.db.migrations.0025_importer_schema_change'
+MIGRATION = 'pulp.server.db.migrations.0027_importer_schema_change'
 
 
 class TestMigration(TestCase):
```

Comment:
(github won't let me leave the comment any lower) Down on line 19, that code no longer gets imported or used by the migration.